src/applications/vaos/new-appointment/components/ReviewPage/PreferredProviderSection.jsx

```diff
@@ -42,12 +52,12 @@ export default function PreferredProviderSection(props) {
               </span>
             </div>
             <div>
-              <Link
-                to={newAppointmentFlow.ccPreferences.url}
+              <VaLink
```

Comment:
let's use `va-link` for consistency
```suggestion
              <va-link
```